### ISSUE TITLE
How to sample the Gumbel noise？

### ISSUE THREAD

**Issue by ysjakking** [EderSantana/gumbel#4]
Hi， thanks for the good work. However I am confused how you sample the  Gumbel noise.
In the sampling function, you just sample the    'Gumbel noise'  from a  'uniform' distribution:
       U = K.random_uniform(K.shape(logits_y), 0, 1)
       y = logits_y - K.log(-K.log(U + 1e-20) + 1e-20) # logits + gumbel noise

I wander if it is right???  Looking forward for the answer~  Thanks

**Comment by ysjakking** [comment]
Never mind , I find the answer.  Your code is right!!   Thanks again!!!

**Comment by EderSantana** [comment]
xD tkx!
